svc.go

```diff
@@ -24,8 +29,11 @@ const P_CIRCUIT = 290
 var (
 	AutoNATServiceDialTimeout   = 15 * time.Second
 	AutoNATServiceResetInterval = 1 * time.Minute
+	AutoNATServiceResetJitter   = 15 * time.Second
 
-	AutoNATServiceThrottle = 3
+	AutoNATServiceThrottle  = 3
+	AutoNATGlobalThrottle   = 30
+	AutoNATMaxPeerAddresses = 16
 )
```

Comment:
Shouldn't we have functional options for these ? If an AutoNAT server has the resources, it could choose to have higher limits.

Reply:
Never-mind. Just saw stebalien's comment above about doing this in a future PR.